daemon/logger/journald/read.go

```diff
@@ -389,7 +393,103 @@ func (s *journald) readLogs(logWatcher *logger.LogWatcher, config logger.ReadCon
 			}
 		}
 	}
-	return
+}
+
+func (s *journald) readRemoteLogs(logWatcher *logger.LogWatcher, config logger.ReadConfig) {
```

Comment:
I think  [the man page for systemd-journal-gatewayd](https://www.freedesktop.org/software/systemd/man/systemd-journal-gatewayd.service.html) is the only document.